src/width.cpp

```diff
@@ -151,10 +151,16 @@ void do_code_width(void)
           (pc->type != CT_SPACE) &&
           is_past_width(pc))
       {
-         if (!split_line(pc))
+         bool split_OK = split_line(pc);
+         if (split_OK)
          {
-            LOG_FMT(LSPLIT, "%s: Bailed on %zu:%zu %s\n",
-                    __func__, pc->orig_line, pc->orig_col, pc->text());
+            LOG_FMT(LSPLIT, "%s(%d): on orig_line=%zu, orig_col=%zu, for %s\n",
+                    __func__, __LINE__, pc->orig_line, pc->orig_col, pc->text());
+         }
+         else
+         {
+            LOG_FMT(LSPLIT, "%s(%d): Bailed on orig_line=%zu, orig_col=%zu, for %s\n",
+                    __func__, __LINE__, pc->orig_line, pc->orig_col, pc->text());
             break;
          }
```

Comment:
This could be made easier like this
<pre>
 const bool split_OK = split_line(pc);
 const char* str = split_OK ? "" : "Bailed ";
 LOG_FMT(LSPLIT, "%s(%d): %son orig_line=%u, orig_col=%u, for %s\n",
                 __func__, __LINE__, str, pc->orig_line, pc->orig_col, pc->text());
 if(split_OK == false)
 {
    break;
 }
</pre>